app/src/main/java/org/eyeseetea/malariacare/DashboardActivity.java

```diff
@@ -176,11 +187,24 @@ private void addTagToLastTab(String tabName){
 
 
     public void initAssess(){
+        isLoadingReview=false;
         unsentFragment = new DashboardUnsentFragment();
         unsentFragment.setArguments(getIntent().getExtras());
         replaceListFragment(R.id.dashboard_details_container, unsentFragment);
     }
 
+    public void restoreAssess(){
+        replaceFragment(R.id.dashboard_details_container, surveyFragment);
+    }
+
+    public void initReview(){
+        int mStackLevel=0;
```

Comment:
@idelcano do you initialise the variable to 0 and in the next line you increase it by one? do we need to do this? what is the purpose of this mStackLevel?